### PR TITLE
docs: add decription to components

### PR DESCRIPTION
**Pull request by TahimiLeonBravo**

### CODE REVIEW

File: src/components/QueryAny/index.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Query from '../Query';
 
+/**
+* The QueryAny predicate takes an array of values. It works exactly the same
+* way as the at operator, but checks whether the fragment matches any of the values in the array.
```
Comment (LeandroTorresSicilia) [inline src/components/QueryAny/index.js]: The QueryAny component ... as the QueryAt component ...
here we are converting this operators to component so talk in term of component

File: src/components/QueryAt/index.js [hunk 2]
```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Query from '../Query';
 
+/**
+* The QueryAt predicate checks that the path matches the described value exactly.
```
Comment (LeandroTorresSicilia) [inline src/components/QueryAt/index.js]: same ^^

File: src/components/QueryDateAfter/index.js [hunk 3]
```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Query from '../Query';
 
+/**
+* The QueryDateAfter predicate checks that the value in the path
```
Comment (LeandroTorresSicilia) [inline src/components/QueryDateAfter/index.js]: same ^^ . Change this in all components

File: src/components/QueryDateAfter/index.js [hunk 4]
```diff
@@ -2,6 +2,10 @@ import React from 'react';
...
+* is after the date value passed into the predicate.
```
Comment (LeandroTorresSicilia) [inline src/components/QueryDateAfter/index.js]: remove the
```into the predicate```
part

File: src/components/QueryDateBefore/index.js [hunk 5]
```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Query from '../Query';
 
+/**
+* The QueryDateBefore predicate checks that the value in the path
+* is before the date value passed into the predicate.
```
Comment (LeandroTorresSicilia) [inline src/components/QueryDateBefore/index.js]: same ^^ remove ```into the predicate``` part. Also remove this for all component. Here does not exist predicate. What they call predicate now is a component and the path is the path prop passed to the component and the value is the value prop.

File: src/components/QueryHour/index.js [hunk 6]
```diff
@@ -2,6 +2,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Query from '../Query';
 
+/**
+* The QueryHour predicate checks that the value in the path occurs within the
+* hour value passed into the predicate.bThis uses the 24 hour system, starting
```
Comment (LeandroTorresSicilia) [inline src/components/QueryHour/index.js]: fix typo: bThis

File: src/components/QueryNot/index.js [hunk 7]
```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Query from '../Query';
 
+/**
+* The QueryNot predicate checks that the path doesn't match the provided value exactly.
+* It takes a single value as the argument.
```
Comment (LeandroTorresSicilia) [inline src/components/QueryNot/index.js]: `It takes a single value as the argument`
Again same a react component does not take an argument it take props and this specifically is explaining the prop value which is already explained.

**Comment by TahimiLeonBravo** [comment]
fixes #21

File: src/components/QueryHour/index.js [hunk 8]
```diff
@@ -2,6 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Query from '../Query';
 
+/**
+* The QueryHour checks that the value in the path occurs within the
+* hour value passed into the predicate.This uses the 24 hour system, starting
```
Comment (LeandroTorresSicilia) [inline src/components/QueryHour/index.js]: remove exactly this:
```into the predicate```

**Pull request merged**